tests/Unit/Informer/Test_MakeTimeInfoFromBuild.py

```diff
@@ -0,0 +1,20 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+from spectre import Informer as Info
+import unittest
+
+
+class TestInformer(unittest.TestCase):
+    def test_spectre_version(self):
+        num_ver = str(Info.spectre_major_version()) + "." + \
+            str(Info.spectre_minor_version()) + "." + \
+            str(Info.spectre_patch_version())
+        self.assertEqual(num_ver, Info.spectre_version())
+
+    def test_unit_test_path(self):
+        self.assertEqual(Info.unit_test_path()[-12:], '/tests/Unit/')
```

Comment:
Could you add a comment about that the twelve is` 5 + 4 + 3` in terms of `tests/unit` things?